dbms/src/Flash/Coprocessor/DAGBlockOutputStream.cpp

```diff
@@ -23,7 +23,7 @@ DAGBlockOutputStream::DAGBlockOutputStream(Block && header_, std::unique_ptr<DAG
 
 void DAGBlockOutputStream::writePrefix()
 {
-    //something to do here?
+    response_writer->prepare(header);
```

Comment:
In previous code, where did the FineGrainedShuffleWriter call `prepare()`?

Reply:
ExchangeSenderBlockInputStream::readPrefix